docs/apis-tools/zeebe-api-rest/tutorial.md

```diff
@@ -0,0 +1,178 @@
+
+:::note
+
+In this tutorial, we will execute arguments to assign and unassign a user to and from a Zeebe user task. You can examine the framework for processing these arguments in the `cli.js` file before getting started.
+
+:::
+
+## Assign a Zeebe user task (POST)
+
+First, let's script an API call to assign a Zeebe user task.
+
+To do this, take the following steps:
+
+1. In the file named `zeebe.js`, outline the authentication and authorization configuration in the first few lines. This will pull in your `.env` variables to obtain an access token before making any API calls:
+
+```javascript
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE,
+};
+```
+
+2. Examine the function `async function assignUser([userTaskKey, assignee])` below this configuration. This is where you will script out your API call.
+3. Within the function, you must first apply an access token for this request, so your function should now look like the following:
+
+```javascript
+async function assignUser([userTaskKey, assignee]) {
+  const accessToken = await getAccessToken(authorizationConfiguration);
+}
+```
+
+4. Using your generated client credentials from [prerequisites](#prerequisites), capture your Zeebe API URL beneath your call for an access token by defining `zeebeApiUrl`:
+
+`const zeebeApiUrl = process.env.ZEEBE_BASE_URL`
+
+5. On the next line, script the API endpoint to assign a Zeebe user task.:
+
+```javascript
+const url = `${ZeebeApiUrl}/user-tasks/${userTaskKey}/assignment`;
+```
+
+6. Configure your POST request to the appropriate endpoint, including an authorization header based on the previously acquired `accessToken`:
+
+```javascript
+const options = {
+  method: "POST",
+  url,
+  headers: {
+    Accept: "application/json",
+    Authorization: `Bearer ${accessToken}`,
+  },
+  data: {
+    // The body contains information about the new assignment.
+    assignee: assignee,
+  },
+};
+```
+
+7. Call the add endpoint, process the results from the API call, and emit an error message from the server if necessary:
+
+```javascript
+try {
+  // Call the add endpoint.
+  const response = await axios(options);
+
+  // Process the results from the API call.
+  if (response.status === 204) {
+    console.log(`User task assigned to ${assignee}.`);
+  } else {
+    // Emit an unexpected error message.
+    console.error("Unable to assign this user!");
+  }
+} catch (error) {
+  // Emit an error from the server.
+  console.error(error.message);
+}
+```
+
+8. In your terminal, run `node cli.js zeebe assign <task id> <assignee@assignee.com>`, where `<task id>` is the Zeebe user task ID, and `<assignee@assignee.com>` is the assignee's email address.
```

Comment:
💡 You could consider having the user assign their own email address if they want to see it in the UIs.

Reply:
🗒️ if you were wondering, like me, what would happen if one assigned a task via the API to a user not in their organization, the answer is:

It will work just fine.

Reply:
Also, I don't have a specific suggestion, but this might also be a good place to specify where they got a Zeebe user task ID from. ("The ID you've captured from Tasklist" or something like that)

Reply:
Adjusted in latest commit 👍